Compile error occurred when using noImplicitAny

I tried `backlog-js` in my Angular project and I got some compile errors.

```
Error: node_modules/backlog-js/dist/backlog.d.ts:178:5 - error TS7010: 'postWatchingListItem', which lacks return-type annotation, implicitly has an 'any' return type.

178     postWatchingListItem(params: any);
        ~~~~~~~~~~~~~~~~~~~~

Error: node_modules/backlog-js/dist/backlog.d.ts:180:5 - error TS7010: 'deletehWatchingListItem', which lacks return-type annotation, implicitly has an 'any' return type.

180     deletehWatchingListItem(watchId: number);
        ~~~~~~~~~~~~~~~~~~~~~~~

Error: node_modules/backlog-js/dist/backlog.d.ts:183:5 - error TS7010: 'getProjectGroupList', which lacks return-type annotation, implicitly has an 'any' return type.

183     getProjectGroupList(projectIdOrKey: string | number);
        ~~~~~~~~~~~~~~~~~~~

Error: node_modules/backlog-js/dist/backlog.d.ts:185:5 - error TS7010: 'postProjectGroup', which lacks return-type annotation, implicitly has an 'any' return type.

185     postProjectGroup(projectIdOrKey: string | number, params: any);
        ~~~~~~~~~~~~~~~~

Error: node_modules/backlog-js/dist/backlog.d.ts:187:5 - error TS7010: 'deleteProjectGroup', which lacks return-type annotation, implicitly has an 'any' return type.

187     deleteProjectGroup(projectIdOrKey: string | number);
```

These errors occurred when I enabled `noImplicitAny` option in `tsconfig.json`. 
And `noImplicitAny` option is enabled by default in Angular projects.

I think it's better to add types to the return value of these functions.

--- a/src/backlog.ts
+++ b/src/backlog.ts
@@ -1101,7 +1101,7 @@ export default class Backlog extends Request {
   /**
    * https://developer.nulab.com/docs/backlog/api/2/add-watching
    */
-  public postWatchingListItem(params: any) {
+  public postWatchingListItem(params: any): Promise<any> {
     return this.post(`watchings`, params);
   }
 
@@ -1115,7 +1115,7 @@ export default class Backlog extends Request {
   /**
    * https://developer.nulab.com/docs/backlog/api/2/delete-watching
    */
-  public deletehWatchingListItem(watchId: number) {
+  public deletehWatchingListItem(watchId: number): Promise<any> {
     return this.delete(`watchings/${watchId}`);
   }
 
@@ -1130,7 +1130,7 @@ export default class Backlog extends Request {
    * https://developer.nulab.com/docs/backlog/api/2/get-project-group-list
    * @deprecated
    */
-  public getProjectGroupList(projectIdOrKey: string | number) {
+  public getProjectGroupList(projectIdOrKey: string | number): Promise<any> {
     console.warn("Deprecated: Use getProjectTeams instead.");
     return this.get(`projects/${projectIdOrKey}/groups`)
   }
@@ -1139,7 +1139,7 @@ export default class Backlog extends Request {
    * https://developer.nulab.com/docs/backlog/api/2/add-project-group
    * @deprecated
    */
-  public postProjectGroup(projectIdOrKey: string | number, params: any) {
+  public postProjectGroup(projectIdOrKey: string | number, params: any): Promise<any> {
     console.warn("Deprecated: Use postProjectTeam instead.");
     return this.post(`projects/${projectIdOrKey}/groups`, params);
   }
@@ -1148,7 +1148,7 @@ export default class Backlog extends Request {
    * https://developer.nulab.com/docs/backlog/api/2/delete-project-group
    * @deprecated
    */
-  public deleteProjectGroup(projectIdOrKey: string | number) {
+  public deleteProjectGroup(projectIdOrKey: string | number): Promise<any> {
     console.warn("Deprecated: Use deleteProjectTeam instead.");
     return this.delete(`projects/${projectIdOrKey}/groups`);
   }

